repl/pom.xml

```diff
@@ -150,6 +150,16 @@
   </build>
   <profiles>
     <profile>
+      <id>hive</id>
+      <dependencies>
+        <dependency>
+          <groupId>org.apache.spark</groupId>
+          <artifactId>spark-hive_${scala.binary.version}</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+      </dependencies>
+    </profile>
+    <profile>
```

Comment:
We can't make this change as I think it means that we will always build hive with the spark assembly, correct?